src/main/resources/validationrules/sscs/sscs-case-role-validation-judicial.drl

```diff
@@ -43,127 +43,106 @@ then
 end;
 
 /*
- * SSCS "panel-doctor" case validation rule.
+ * SSCS "tribunal-member-1" case validation rule.
  */
 
-rule "sscs_panel_doctor_create_case_roles"
+rule "sscs_tribunal_member_1_create_case_roles"
 when
     $f:  FeatureFlag(status && flagName == FeatureFlagEnum.SSCS_WA_1_0.getValue())
     $ca: CaseAllocatorApproval(
                  roleAssignment.status == Status.CREATE_REQUESTED,
                  roleAssignment.attributes["jurisdiction"].asText() == "SSCS",
                  roleAssignment.attributes["caseType"].asText() == "Benefit",
-                 roleAssignment.roleName in ("panel-doctor"))
+                 roleAssignment.roleName in ("tribunal-member-1"))
          ExistingRoleAssignment(
                  actorId == $ca.getRoleAssignment().actorId,
                  roleType == RoleType.ORGANISATION,
-                 roleName in ("fee-paid-medical", "medical"),
+                 roleName in ("fee-paid-medical", "medical", "fee-paid-disability", "fee-paid-financial"),
                  attributes["jurisdiction"] != null && attributes["jurisdiction"].asText() == "SSCS")
 then
+    $ca.getRoleAssignment().setAttribute("workTypes", "hearing_work,priority");
```

Comment:
workType can only be populated from the ORM and it should be applicable only for Org roles not for case roles. Please check with @nitinsahu1

Reply:
removed worktypes for case roles